test/bigtable/index.js

```diff
@@ -110,6 +110,24 @@ describe('Bigtable', function() {
       fakeUtil.normalizeArguments = normalizeArguments;
     });
 
+    it('should leave the original options unaltered', function() {
```

Comment:
Can we just see if the same object the user provides (or at least the same properties they defined) make it to `GrpcService.call(this, config, options)`. Sorry if this test is doing that already and I didn't see the connection.